Makefile

```diff
@@ -0,0 +1,16 @@
+HELM != helm
+
+gen-expected:
+	${HELM} template --namespace=default test . > tests/expected.yaml
+	sed -i 's/[[:blank:]]\+$$//g'  tests/expected.yaml
+	${HELM} template --namespace=default --values=tests/recommend.yaml test . > tests/recommend-expected.yaml
+	sed -i 's/[[:blank:]]\+$$//g'  tests/recommend-expected.yaml
+
+start-local:
+	k3d cluster delete test-cluster || true
```

Comment:
seems good, but I wonder what happens if we have many jobs at the same time on the same github actions instance. Is this good encapsulated?

Reply:
I hope so :-)